src/stop.sh

```diff
@@ -200,7 +200,7 @@ if [ $stop_all -eq 1 ]; then
     fi
 
     # killing processes
-    to_kill="$ceph_osd ceph-mon ceph-mds ceph-mgr radosgw lt-radosgw apache2 ganesha.nfsd cephfs-top"
+    to_kill="$ceph_osd ceph-mon ceph-mds ceph-mgr radosgw lt-radosgw apache2 ganesha.nfsd cephfs-top cephfs-mirror"
```

Comment:
Let's do the same for rbd-mirror while at it ;)
```suggestion
    to_kill="$ceph_osd ceph-mon ceph-mds ceph-mgr radosgw lt-radosgw apache2 ganesha.nfsd cephfs-top cephfs-mirror rbd-mirror"
```